### DISCUSSION TITLE
Gemini Endpoint does not have auto title generation

### DISCUSSION THREAD

**Discussion by Astricaelus** [danny-avila/LibreChat#3017] [Troubleshooting] [answered]
In the docs, I didn't see any environment variables that specify the titling model for gemini. And when I chat with gemini, libre chat doesn't automatically generate the title.

**Comment by danny-avila** [comment, marked as answer]
See the open issues/pull requests on this:

https://github.com/danny-avila/LibreChat/issues/1422
https://github.com/danny-avila/LibreChat/pull/2983
